src/app/core/services/issue.service.ts

```diff
@@ -296,6 +296,7 @@ export class IssueService {
         issuesAPICallsByFilter.push(
           this.githubService.fetchIssuesGraphql(new RestGithubIssueFilter({ creator: this.userService.currentUser.loginId }))
         );
+
```

Comment:
Should this empty line be removed to make the code uniform with the other switch cases?

Reply:
Sure thing